metron-platform/metron-common/src/main/java/org/apache/metron/common/message/JSONFromPosition.java

```diff
@@ -40,10 +40,12 @@ public JSONFromPosition(int position) {
 
   @Override
   public JSONObject get(Tuple tuple) {
+    String s = null;
     try {
-      return (JSONObject) parser.get().parse(new String(tuple.getBinary(position), "UTF8"));
+      s =  new String(tuple.getBinary(position), "UTF8");
```

Comment:
Can we just use StandardCharsets.UTF_8 then, and not even worry about the string being right?  I'm not sure where that alias is, so I'm not convinced it's guaranteed (although a source proving me wrong would also be acceptable)